src/lib.rs

```diff
@@ -47,12 +47,12 @@
 //! ```
 
 extern crate libc;
-#[cfg(feature = "tokio")]
+#[cfg(feature = "capture-stream")]
```

Comment:
These extern crates aren't necessary anymore now that the code is being switched to 2018 edition.